solr/bin/solr

```diff
@@ -2084,14 +2084,14 @@ else
   REMOTE_JMX_OPTS=()
 fi
 
-# Enable java security manager (limiting filesystem access and other things)
-if [ "$SOLR_SECURITY_MANAGER_ENABLED" == "true" ]; then
+# Disable java security manager (allowing filesystem access and other things)
+if [ "$SOLR_SECURITY_MANAGER_ENABLED" == "false" ]; then
```

Comment:
I think we may prefer the bashism of `if [ "${SOLR_SECURITY_MANAGER_ENABLED:-true}" == "true" ]` for consistency elsewhere in the script. it makes it clear that if its undefined, it will default to true.

the windows file `solr.cmd` needs to be changed too. Currently this pull request doesn't change anything for windows. for windows CMD, I think it is clearest to just put a `IF NOT DEFINED` before the check. see that linked patch for an example.

Reply:
should be resolved?